pkg/api/customization/catalog/validator.go

```diff
@@ -24,11 +25,15 @@ func Validator(request *types.APIContext, schema *types.Schema, data map[string]
 			u.Scheme = strings.ToLower(u.Scheme) // git commands are case-sensitive
 			data["url"] = u.String()
 		}
-		return nil
 	} else if request.Method == http.MethodPost {
 		return httperror.NewAPIError(httperror.MissingRequired, "Catalog URL not specified")
 	}
 
+	if helmVersion, ok := data["helmVersion"]; ok {
+		if validateHelm3Version(strings.ToLower(helmVersion.(string))) {
+			return httperror.NewAPIError(httperror.InvalidBodyContent, "Invalid helm 3 version")
```

Comment:
I'm confused here. If your validate functions returns true, then the helm 3 version is invalid?